build.gradle

```diff
@@ -567,15 +567,19 @@ idea {
 
 // this allows ./gradlew run
 bootJar {
-
   manifest {
     attributes 'Implementation-Title': project.name,
             'Implementation-Version': project.version
   }
+  duplicatesStrategy = DuplicatesStrategy.EXCLUDE
   archiveName = 'civil-sdt.jar'
+  from {
```

Comment:
if we have this one inplace, I don't think below is needed

  "consumers/build/libs/civil-sdt-consumers-${version}.jar",
      "dao/build/libs/civil-sdt-dao-${version}.jar",
      "dao-api/build/libs/civil-sdt-dao-api-${version}.jar",
      "domain/build/libs/civil-sdt-domain-${version}.jar",

Reply:
can you please check this one